src/App.tsx

```diff
@@ -1,35 +1,94 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+/*
+import { useState } from "react"
+import reactLogo from "./assets/react.svg"
+import viteLogo from "/vite.svg"
+import "./App.css"
+*/
+import { useState } from "react";
 
-function App() {
-  const [count, setCount] = useState(0)
+const Create = () => {
```

Comment:
Let's write our components with the function keyword syntax.

```suggestion
function Create()  {
```